modules/plugin/jdbc/jdbc-oracle/src/test/java/org/geotools/data/oracle/OracleNGDataStoreFactoryOnlineTest.java

```diff
@@ -85,6 +96,40 @@ public void testGeometryMetadata() throws IOException {
         }
     }
 
+    @Test
+    public void testTimeoutProps() throws IOException, ReflectiveOperationException {
+        OracleNGDataStoreFactory factory = new OracleNGDataStoreFactory();
+        Properties db = FixtureUtilities.loadFixture("oracle");
+        Map<String, Object> params = new HashMap<>();
+        params.put(HOST.key, db.getProperty(HOST.key));
+        params.put(DATABASE.key, db.getProperty(DATABASE.key));
+        params.put(SCHEMA.key, db.getProperty(SCHEMA.key));
+        params.put(PORT.key, db.getProperty(PORT.key));
+        params.put(USER.key, db.getProperty(USER.key));
+        params.put(PASSWD.key, db.getProperty("password"));
+        params.put(DBTYPE.key, "oracle");
+        params.put(LOGIN_TIMEOUT.key, 1);
```

Comment:
I'd keep these values as high as possible, just to make sure we don't get timeouts in Github actions, which might be occasionally very slow. In the end, the test is just checking they have been injected, you're not actually trying to trigger a timeout.